code/game/objects/items/nemesis_items.dm

```diff
@@ -0,0 +1,458 @@
+			return ..()
+
+		gloves.lose_charge(NEMESIS_CHARGE_PER_MINE)
+
+	overloaded = !overloaded
+	to_chat(user, "<span class='notice'>You [overloaded ? "overload [src]'s detection system" : "return [src]'s circutry back to the normal state"].</span>")
+	icon_state = "[overloaded ? "nemesis_trap" : "nemesis_trap_inactive"]"
+
+/obj/item/nemesis_mine/attack_hand(mob/living/user, list/modifiers)
+	if(!active)
+		return ..()
+
+	to_chat(user, "<span class='warning'>You attempt to disarm [src]...</span>")
+	var/disarm_time = 15 SECONDS
+	if(ishuman(user))
+		var/mob/living/carbon/human/disarmer = user
+		if(istype(disarmer.get_item_by_slot(ITEM_SLOT_GLOVES), /obj/item/clothing/gloves/rapid/nemesis))
+			disarm_time = 1 SECONDS
+
+	if(!do_after(user, disarm_time, src))
+		return
+	active = FALSE
+	icon_state = initial(icon_state)
+	update_icon()
+	anchored = FALSE
+	to_chat(user, "<span class='notice'>You successfully disarm [src].</span>")
+
+/obj/item/nemesis_mine/attackby(obj/O, mob/user, params)
+	if(!active)
+		return ..()
+
+	if(!istype(O, /obj/item/screwdriver))
+		return ..()
+
+	to_chat(user, "<span class='warning'>You attempt to disarm [src]...</span>")
+	var/disarm_time = 3 SECONDS
+	if(ishuman(user))
+		var/mob/living/carbon/human/disarmer = user
+		if(istype(disarmer.get_item_by_slot(ITEM_SLOT_GLOVES), /obj/item/clothing/gloves/rapid/nemesis))
+			disarm_time = 1 SECONDS
+
+	if(!do_after(user, disarm_time, src))
+		return
+
+	active = FALSE
+	icon_state = initial(icon_state)
+	update_icon()
+	anchored = FALSE
+	to_chat(user, "<span class='notice'>You successfully disarm [src].</span>")
+
+
+/obj/item/nemesis_mine/proc/trigger(mob/living/victim)
+	var/turf/cur_loc = get_turf(src)
+	new /obj/effect/temp_visual/nemesis_circle(cur_loc)
+
+	if(!victim)
+		victim = locate() in cur_loc
+
+	if(victim)
+		victim.apply_damage(75, STAMINA)
+		to_chat(victim, "<span class='userdanger'>You trigger [src] and it explodes in shower of sparks!</span>")
+
+	playsound(cur_loc, 'sound/magic/disable_tech.ogg', 50, TRUE)
+
+	if(glasses && ishuman(glasses.loc))
+		var/mob/living/carbon/human/owner = glasses.loc
+		if(owner.get_item_by_slot(ITEM_SLOT_EYES) != glasses)
+			return
+		glasses.say("Attention: Tactical energy mine in [get_area(cur_loc)] has detonated.")
+
+	qdel(src)
+
+/obj/item/nemesis_mine/proc/trigger_overload()
+	var/turf/cur_loc = get_turf(src)
+	new /obj/effect/temp_visual/nemesis_circle/huge(cur_loc)
+	playsound(cur_loc, 'sound/magic/disable_tech.ogg', 50, TRUE)
+	for(var/mob/living/victim in range(1, cur_loc))
+		to_chat(victim, "<span class='userdanger'>You are hit by an energy blast!</span>")
+		victim.apply_damage(55, STAMINA)
+	qdel(src)
+
+/obj/item/nemesis_mine/throw_at(atom/target, range, speed, mob/thrower, spin=1, diagonals_first = 0, datum/callback/callback, quickstart = TRUE)
+	. = ..()
+	icon_state = "trap_small"
+
+/obj/item/nemesis_mine/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
```

Comment:
I forget, does this run if the mine doesn't actually hit anything and just stops somewhere in the middle of a hallway?

Reply:
also, can you just use these as a normal sec officer? the only things that seems to require the suit are overloading them (although your suited up friend can just overload them before handing them to you) and getting remote feedback that they've been triggered

Reply:
Yes, this triggers if it hits the turf as well. And yes, I should probably block them from activating if the thrower wasn't wearing an active Nemesis suit